CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ is more like an open wiki than a standard guarded open source project.
 
 There are a few basic ground-rules for contributors:
 
-1. **No `--force` pushes** or modifying the Git history in any way.
+1. **No `--force` pushes** or modifying the Git history in any way, with the exception of rebasing or squashing pull requests on GitHub.
```

Comment:
Maybe

**No `--force` pushes** or modifying the Git history unless rebasing or squashing pull requests on GitHub.